CMakeLists.txt

```diff
@@ -455,10 +464,6 @@ IF(PLUS_USE_OPTITRACK)
   LIST(APPEND PLUSBUILD_QT_COMPONENTS Multimedia)
 ENDIF()
 
-IF(PLUS_USE_CLARIUS_OEM)
-  LIST(APPEND PLUSBUILD_QT_COMPONENTS Bluetooth)

```

Comment:
Is this no longer required?

Reply:
Correct, the device uses the native C++/WinRT interface to manage BLE connections. This is a relic of a previous Qt based solution.